sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1777,6 +1777,23 @@ object functions {
   def round(e: Column, scale: Int): Column = withExpr { Round(e.expr, Literal(scale)) }
 
   /**
+   * Returns the value of the column `e` rounded to 0 decimal places with HALF_EVEN round mode.
+   *
+   * @group math_funcs
+   * @since 2.0.0
+   */
+  def bround(e: Column): Column = bround(e, 0)
```

Comment:
No, please don't do JIRAs that way.  A JIRA that just refers to a PR (or a PR description that just refers to a JIRA number) is nearly pointless and very annoying.  Always include a description in the JIRA motivating *why* a change is needed or desired, and a description of *what* was changed in the PR.